src/table/table-model.class.ts

```diff
@@ -52,6 +52,11 @@ export class TableModel implements PaginationModel {
 	dataChange = new EventEmitter();
 	rowsSelectedChange = new EventEmitter<number>();
 	rowsExpandedChange = new EventEmitter<number>();
+	/**
+	 * Gets emitted when `selectAll` is called. Emits false if all rows are deselected and true if
+	 * all rows are selected.
+	 */
+	selectAllChange = new EventEmitter<boolean>();
```

Comment:
Should be `Subject<boolean>()` - `EventEmitter` isn't intended to be used outside of component classes